<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bookInfo</class>
 <widget class="QDialog" name="bookInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="titleSubLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Название</string>
   </property>
  </widget>
  <widget class="QLabel" name="authorSubLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Автор</string>
   </property>
  </widget>
  <widget class="QLabel" name="pageCountSubLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Количество страниц</string>
   </property>
  </widget>
  <widget class="QLabel" name="costLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Цена</string>
   </property>
  </widget>
  <widget class="QLabel" name="pageCountSub2Label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttons">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>166</width>
     <height>21</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="giveBookSubLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Выдать книгу:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="readerSelector">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>201</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="returnBookButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Вернуть книгу</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="titleEditor">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="authorEditor">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="costEditor">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
